src/qiboconnection/api.py

```diff
@@ -387,15 +387,11 @@ def execute(  # pylint: disable=too-many-locals, disable=too-many-branches
             raise ValueError("No devices were selected for execution.")
         if isinstance(circuit, Circuit):
             circuit = [circuit]
-
```

Comment:
nice, I did this weird thing because otherwise linters crashed. Happy to see this deleted :)